.github/workflows/ci-optimized.yml

```
@@ -0,0 +1,848 @@
+            --cache-to type=gha,mode=max \
+            ./backend
+
+          echo "📤 Pushing Docker image to Artifact Registry..."
+          docker push "${IMAGE_SHA}"
+          docker push "${IMAGE_LATEST}"
+
+          echo "IMAGE_SHA=${IMAGE_SHA}" >> $GITHUB_OUTPUT
+          echo "IMAGE_LATEST=${IMAGE_LATEST}" >> $GITHUB_OUTPUT
+
+      - name: Add build summary
+        run: |
```

actionlint (1.7.8)

got unexpected character ':' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)